requirements/base.txt

```diff
@@ -0,0 +1,7 @@
+pip==8.1.2
+bumpversion==0.5.3
+wheel==0.29.0
+watchdog==0.8.3
+django==1.9.7
+six
```

Comment:
I think there may also be a version of `six` bundled into `django` that you can use instead.